.gitignore

```diff
@@ -0,0 +1 @@
+image-builder/nsdax
```

Comment:
Nit: Missing newline. I'm guessing you're an emacs user ;)

Reply:
yup :smile: - fixed thanks